CONTRIBUTING.md

```diff
@@ -591,13 +592,22 @@ scenarios in this guide.
 
 1. Put the name of the exercise in the subject line of the commit.
    E.g. `hamming: Add test case for strands of unequal length`
+1. The subject line should be a one-sentence summary. This is something that
```

Comment:
I think "This is something that should not include the word *and*" could be simplified to something like "It should not include the word *and*"